src/main.rs

```diff
@@ -134,11 +121,6 @@ fn main() -> Result<(), aes_gcm_siv::Error> {
     let evals = (0..20)
```

Comment:
why 20?

Reply:
I stole that part of the code from `https://github.com/arkworks-rs/sumcheck/blob/f4d971ee02a3116442bf393c305a734933b20dde/src/ml_sumcheck/protocol/verifier.rs#L298C13-L298C13
`. 20 was used in one of the tests.